app/boards/arm/s40nc/Kconfig.board

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2021 Kyle McCreery
```

Comment:
Would you feel comfortable using `The ZMK Contributors` for copyrights? It helps us avoid churn, but it's your choice.

Reply:
That is fine if preferred. I was basing my change from the clarification of licensing that was forced on other similar projects so I went with most specific. I will update accordingly.